### DISCUSSION TITLE
[Feature request] Click to copy for simple `codeblocks`

### DISCUSSION THREAD

**Discussion by couleurm** [squidfunk/mkdocs-material#6393] [Ideas]
I know it's already a thing for the tripe \`\`\` one but it'd be cool to be able to talk about a path like `%TEMP%\mkdocs_7uplsql4` and easily be able to click it to copy to clipboard and it could be syntaxed like

```
`useful thing to copy`{click-to-copy}
```

**Comment by squidfunk** [comment]
You can already do this by using `data-clipboard-text`:

``` md
`copy me`{ data-clipboard-text="copy me" }
```

Yes, you  have to duplicate the text to copy, but it allows you to specify exactly what to copy. With some [additional CSS](https://squidfunk.github.io/mkdocs-material/customization/#additional-css), you can also make it more apparent that the text can be copied:

``` css
.md-typeset [data-clipboard-text] {
  cursor: pointer;
}
```

**Comment by couleurm** [comment]
perfect 🥰🥰🥰
